apps/kitty-krew/Dockerfile

```
@@ -2,35 +2,67 @@ FROM oven/bun:latest AS builder
+RUN echo "Copying files from builder stage"
 COPY --from=builder /app/.output ./output
+RUN echo "Output directory copied - contents:"
+RUN ls -la ./output || echo "output directory not found!"
+
 COPY --from=builder /app/node_modules ./node_modules
+RUN echo "Node modules copied - checking contents:"
+RUN ls -la node_modules | head -n 5 || echo "node_modules directory not found!"
+
 COPY --from=builder /app/package.json ./package.json
+RUN echo "Package.json copied:"
+RUN cat package.json | head -n 10 || echo "package.json not found!"
```

Hadolint (2.12.0)

[warning] Set the SHELL option -o pipefail before RUN with a pipe in it. If you are using /bin/sh in an alpine image or if your shell is symlinked to busybox then consider explicitly setting your SHELL to /bin/ash, or disable this check

(DL4006)